engine/tests/story/test_choice_availability.py

```
@@ -177,14 +204,18 @@ def test_failing_predicate_reason_code(self) -> None:
 
         assert fragments and fragments[0].unavailable_reason == "guard_failed_or_unavailable"
 
-    def test_failing_predicate_blocker_type_is_edge(self) -> None:
+    def test_failing_predicate_emits_portable_blocker(self) -> None:
         graph, start, _end, _action = _graph_with_choice(guard_expr="False")
```

Ruff (0.15.15)

[warning] Unpacked variable `graph` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)